src/Microsoft.AspNet.Session/CookieSecureOption.cs

```diff
@@ -0,0 +1,39 @@
+﻿// Copyright (c) Microsoft Open Technologies, Inc. All rights reserved.
```

Comment:
BTW we just changed the copyright noticed to:

`Copyright (c) .NET Foundation. All rights reserved.`